docs/rules/0134/update-mask-optional-behavior.md

```diff
@@ -0,0 +1,72 @@
+---
+rule:
+  aip: 134
+  name: [core, '0134', update-mask-optional-behavior]
+  summary: Standard Update `update_mask` field must be `OPTIONAL`.
+permalink: /134/update-mask-optional-behavior
+redirect_from:
+  - /0134/update-mask-optional-behavior
+---
+
+# Update methods: Mask field expected behavior
+
+This rule enforces that the `update_mask` field of a Standard `Update` request
+uses `google.api.field_behavior = OPTIONAL`, as mandated in [AIP-134][].
+
+## Details
+
+This rule looks at any field named `update_mask` that's in a `Update*Request`
+and complains if it the field is not annotated with
+`google.api.field_behavior = OPTIONAL`.
+
+## Examples
+
+**Incorrect** code for this rule:
```

Comment:
This shows a field with an incorrect field_behavior - what if the behavior is missing entirely? (My guess is that this is covered by a different rule.)

Reply:
(I see there's a test for this - maybe it could be included as another incorrect example? Not sure if we ever have multiple incorrect examples.)

Reply:
We don't typically include multiple incorrect examples, no, but yes it is handled.